js/npm/sequelize/DataTypes.hx

```diff
@@ -28,14 +28,14 @@ class DataTypes {
 	}
 	/**
 	 * A 32 bit integer. 
-	 * Available properties: `length`, `unsigned:Bool`, `zerofill:Bool`
+	 * Available properties: `length:Int`, `unsigned:Bool`, `zerofill:Bool`
 	 */
 	static public inline function INTEGER(?options:{}):DataTypes {

```

Comment:
Oh I just saw the datatypes methods are on `Sequelize` in reality...
I guess you can just `implements npm.Require('sequelize')` here and remove `@:native('DataTypes')` (that was useless anyway)